mindsdb_native/libs/phases/model_analyzer/model_analyzer.py

```diff
@@ -46,6 +46,17 @@ def run(self):
             backend=self.transaction.model_backend
         )
 
+        # @TODO Limiting to 4 as to not kill the GUI, sample later (or maybe only select latest?)
+        if self.transaction.lmd['tss']['is_timeseries'] and len(normal_predictions[output_columns[0]]) < pow(10,4):
+            self.transaction.lmd['test_data_plot'] = {}
+            for output_column in output_columns:
+                self.transaction.lmd['test_data_plot'][col] = {
+                    'real': list(validation_df[output_column])
```

Comment:
Maybe we ought to return an array of real values if `nr_predictions > 1` in this key? It's redundant information that could be obtained in the GUI, though.